pages/compiling/installation-reqs.rst

```diff
@@ -154,18 +159,10 @@ Conda Requirements
   containing ``sphinx``, ``doxygen``, ``graphviz``, ``pip`` as well as all relevant
   items listed for the ``pip:`` entry in ``environment.yaml``.
 
-* Now you can build and test Cantera with
-
-  .. code:: bash
-
-     scons build
-     scons test
-
-* To install Cantera, use the command
-
-  .. code:: bash
-
-     scons install
```

Comment:
Any reason to remove this?

Reply:
Yes. If you look at the actual page, the next step is to download the source :wink: ... https://testing.cantera.org/compiling/installation-reqs.html#conda-requirements